test/deploy-prepare-and-test-pipeline.sh

```diff
@@ -0,0 +1,157 @@
+SOURCE_STORAGE_ACCOUNT_NAME=${8:-${SOURCE_STORAGE_ACCOUNT_NAME:-"stellarciresources"}}
+SOURCE_STORAGE_ACCOUNT_RG_NAME=${9:-${SOURCE_STORAGE_ACCOUNT_RG_NAME:-"ci-resources"}}
+SYNAPSE_RESOURCE_GROUP_NAME=${10:-${SYNAPSE_RESOURCE_GROUP_NAME:-"${ENV_CODE}-pipeline-rg"}}
+SYNAPSE_WORKSPACE_NAME=${11:-${SYNAPSE_WORKSPACE_NAME:-"${ENV_CODE}-pipeline-syn-ws"}}
+BATCH_JOB_NAME=${12:-${BATCH_JOB_NAME:-"${ENV_CODE}-data-cpu-pool"}}
+AOI_PARAMETER_VALUE=${13:-${AOI_PARAMETER_VALUE:-"-117.063550 32.749467 -116.999386 32.812946"}}
+PIPELINE_FILE_NAME=${14:-${PIPELINE_FILE_NAME:-"E2E Custom Vision Model Flow.json"}}
+PARAMETER_FILE_PATH=${15:-${PARAMETER_FILE_PATH:-"${PRJ_ROOT}/test/${PIPELINE_NAME}-parameters.json"}}
+SUBSTITUTE_PARAMS=${16:-${SUBSTITUTE_PARAMS:-"true"}}
+
+set -ex
+
+if [[ -z "$ENV_CODE" ]]
+  then
+    echo "Environment Code value not supplied"
+    exit 1
+fi
+
+if [[ -z "$DESTINATION_STORAGE_ACCOUNT_NAME" ]]; then
+    DESTINATION_STORAGE_ACCOUNT_NAME=$(az storage account list --resource-group $DESTINATION_STORAGE_ACCOUNT_RG_NAME --query [0].name -o tsv)
+fi
+if [[ -z "$STORAGE_ACCOUNT_KEY" ]]; then
+    STORAGE_ACCOUNT_KEY=$(az storage account keys list --resource-group ${DESTINATION_STORAGE_ACCOUNT_RG_NAME} --account-name ${DESTINATION_STORAGE_ACCOUNT_NAME} --query "[0].value" --output tsv)
+fi
+if [[ -z "$SOURCE_STORAGE_ACCOUNT_KEY" ]]; then
+    SOURCE_STORAGE_ACCOUNT_KEY=$(az storage account keys list --resource-group ${SOURCE_STORAGE_ACCOUNT_RG_NAME} --account-name ${SOURCE_STORAGE_ACCOUNT_NAME} --query "[0].value" --output tsv)
+fi
+
+# Create container in destination storage account
+echo "Creating container ${DESTINATION_CONTAINER_NAME} on storage account ${DESTINATION_STORAGE_ACCOUNT_NAME}"
+az storage container create --name ${DESTINATION_CONTAINER_NAME} --account-name ${DESTINATION_STORAGE_ACCOUNT_NAME} --account-key ${STORAGE_ACCOUNT_KEY}
+
+
+# Upload test data to destination storage account container
+echo "Copying resources to storage container ${DESTINATION_CONTAINER_NAME} on storage account ${DESTINATION_STORAGE_ACCOUNT_NAME}"
+az storage blob copy start \
+  --destination-blob raw/sample_4326.tif \
+  --destination-container ${DESTINATION_CONTAINER_NAME} \
+  --account-name ${DESTINATION_STORAGE_ACCOUNT_NAME} \
+  --account-key ${STORAGE_ACCOUNT_KEY} \
+  --source-container ${SOURCE_CONTAINER_NAME} \
+  --source-account-name ${SOURCE_STORAGE_ACCOUNT_NAME} \
+  --source-account-key ${SOURCE_STORAGE_ACCOUNT_KEY} \
+  --source-blob raw/sample_4326.tif
+
+az storage blob copy start \
+  --destination-blob config/config.json \
+  --destination-container ${DESTINATION_CONTAINER_NAME} \
+  --account-name ${DESTINATION_STORAGE_ACCOUNT_NAME} \
+  --account-key ${STORAGE_ACCOUNT_KEY} \
+  --source-container ${SOURCE_CONTAINER_NAME} \
+  --source-account-name ${SOURCE_STORAGE_ACCOUNT_NAME} \
+  --source-account-key ${SOURCE_STORAGE_ACCOUNT_KEY} \
+  --source-blob config/config.json
```

Comment:
Same here. Remove hardcoded file names. May be store it in a centralized configuration file.